snapshot/modify_space.js

```diff
@@ -0,0 +1,146 @@
+const { ethers } = require('ethers');
+const snapshot = require('@snapshot-labs/snapshot.js');
```

Comment:
There not being a Python alternative to "snapshot.js" is pretty frustrating. Are we in too deep with the discord bot to move to TypeScript? Maybe we should discuss an alternative to snapshot. Overall, kind of unhappy with it

Reply:
> There not being a Python alternative to "snapshot.js" is pretty frustrating. Are we in too deep with the discord bot to move to TypeScript? Maybe we should discuss an alternative to snapshot. Overall, kind of unhappy with it

Quite frustrating indeed. Am not sure about being in too deep, is there an alternative that comes to mind? Am happy to explore other options if it makes sense for us to do so.